LICENSES/README

```diff
@@ -0,0 +1,60 @@
+The systemd project uses a single line reference to Unique License Identifiers
+in source files as defined by the Linux Foundation's SPDX project
+(https://spdx.org/).
+
+The current set of valid, predefined SPDX identifiers can be found on the SPDX
+License List at https://spdx.org/licenses/.
+
+The 'license/' directory contains all the licenses used by the sources included
+in the systemd project source tree.
+
+Unless otherwise noted, the systemd project sources are licensed under the terms
+and conditions of the "GNU Lesser General Public License v2.1 or later".
+
+The following exceptions apply:
+
+ * the udev sources are licensed under GPL-2.0-or-later
```

Comment:
it's more complicated actually, some files (`src/udev/udevadm-hwdb.c`) are lgpl2.1+ too there